### PR TITLE
Address issue #2 to fix the inconsistence results.

### PR DESCRIPTION
**Pull request by zztin**

### CODE REVIEW

File: atta/cleaner/title.py [hunk 1]
```diff
@@ -11,21 +11,20 @@ def cat_title(title):
...
+    pattern_dic['Head'] = "(?i)C.O|chief|lead|chair|director|長|總|founder"
+    pattern_dic['Manager'] = "(?i)manager|[^助]理|pm"
+    pattern_dic['Engineer'] = "(?i)[engineer]{6,}|engr|develop|code|software|工程師|碼|程式|資訊|program|" \
+                              "軟體|設計|IT|Analysts|SW|AP|PG|F2E|DevOps|architect|^R.*D$|開發|bug|hacker"
+    #'設計'有點不精確...
+    pattern_dic['Student'] = "學生|(?i)student"
+    pattern_dic['Academia'] = "(?i)phd|博|postdoc|research|研究|PI|professor|教授"
+    pattern_dic['Data Scientist'] = "(?i)data|資料|使用|經驗|分析|scientist"
+    pattern_dic['Consultant'] =  "(?i)consultant|顧問"
+
```
Comment (tai271828) [inline atta/cleaner/title.py]: I like the new coding style. More clear to me.

File: atta/cleaner/title.py [hunk 2]
```diff
@@ -36,4 +35,4 @@ def cat_title(title):
 
 if __name__ == "__main__":
     import doctest
-    doctest.testmod()
+    doctest.testmod()
```
Comment (tai271828) [inline atta/cleaner/title.py]: No newline.

File: atta/cleaner/title.py (hunk 1 above)
Comment (zztin) [reply atta/cleaner/title.py]: Great

Comment (zztin) [reply atta/cleaner/title.py]: Does it fix the issue on your machine as well?

**Comment by tai271828** [comment]
Looks good to me. Only minor fix suggestions. However it will be much better if the commit message containing the issue number. I will close this PR and wait for another. Thanks.

**Comment by tai271828** [comment]
Ok I saw the commit message with issue #. Although it is not perfect, it should still be good enough for this time.

File: atta/cleaner/title.py (hunk 1 above)
Comment (tai271828) [reply atta/cleaner/title.py]: Yes. Thanks for the patch.

**Pull request merged**